pkgs/by-name/sh/shh/package.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, python3
+, strace
+, systemd
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "shh";
+  version = "2023.12.16";
+
+  src = fetchFromGitHub {
+    owner = "desbma";
```

Comment:
Why are we not using it from the `synacktiv` Github account? Yknow, the one that made the blog and is linked in `meta.homepage`.

Reply:
Oops, I used the wrong URL for meta.homepage. I believe desbma is the original author (as stated in project README) and synactiv forked it (Synactiv is a French IT company, maybe desbma is working there?).